### DISCUSSION TITLE
Possibilité d'utilisation en version 'pleine écran'

### DISCUSSION THREAD

**Discussion by atarixGB** [MTESSDev/utd-webcomponents#63] [Q&A] [answered]
Bonjour à l'équipe,

Nous sommes en développement d'une plateforme pour utilisation interne et nous nous demandions s'il est possible d'utiliser les composantes en version plein écran.

Plus précisément, avoir un contenu occupant la largeur de l'écran (section verte) avec des marges plus petite (sections roses) ![image](https://github.com/MTESSDev/utd-webcomponents/assets/56455298/4f1bb2fd-9cad-443b-9983-be02d9abb3a7)

au lieu d'un contenu fixe (version actuelle)
![image](https://github.com/MTESSDev/utd-webcomponents/assets/56455298/d0a2bee2-fece-4cc5-993a-eb8021b02d3e)

Merci!

**Comment by j0ber** [comment, marked as answer]
Oui c'est possible. https://lively-cliff-018a9540f.azurestaticapps.net/gabaritpleinelargeur
